CONTRIBUTING.md

```diff
@@ -54,6 +54,55 @@ The ADF-Rules are as follows:
 * Class names cannot start with Alfresco/Activiti/adf for the same reason as above
 * scss is mandatory. All the classes need to have the `adf-` prefix
 
+## Styles
+
+Avoid adding css variables with names related to components:
```

Comment:
maybe all this part is better here https://github.com/Alfresco/alfresco-ng2-components/blob/develop/docs/user-guide/theming.md

Reply:
fixed